### PR TITLE
chore: Add support for Scala 2.13.10

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: project/V.scala [hunk 1]
```diff
@@ -72,6 +72,8 @@ object V {
     "2.13.2",
     "2.13.3",
     "2.13.4",
+    // has a binary compatibility issue
+    "2.13.9",
```
Comment (kpodsiad) [inline project/V.scala:76]: It'll good to somehow inform users about that binary compatibility issue

Comment (tgodzik) [reply project/V.scala]: We can do it in another PR

Comment (tanishiking) [reply project/V.scala]: > we'll deprecate it separately

Do you mean, we'll drop 2.13.9 support from Metals? I think showing some kind of warning is enough.
(anyway, we can do that in another PR :+1:)

**Comment by tgodzik** [comment]
I will go ahead and merge this, since it's needed for the Scala release

**Pull request merged**